bioptim/dynamics/configure_problem.py

```diff
@@ -102,6 +102,7 @@ def torque_driven(
         nlp,
         with_contact: bool = False,
         implicit_dynamics: bool = False,
+        semi_implicit_dynamics: bool = False,
```

Comment:
Just to make sure it is not a typo, in the PR name, you called it semi_explicit, which one it is?